Tests/MapboxNavigationTests/NavigationMapViewTests.swift

```diff
@@ -200,7 +204,7 @@ class NavigationMapViewTests: TestCase {
     func testOverriddenStreetsRouteClassMotorwayMixedCongestionLevels() {
         let route = loadRoute(from: "route-with-mixed-road-classes")
         var congestions = route.congestionFeatures()
-        XCTAssertEqual(congestions.count, 5)
+        XCTAssertEqual(congestions.count, 1)
```

Comment:
In one route, each leg has one congestion attribute feature. Each attribute feature contains the UIColor and the associate color change start point fraction in this leg distance.